apps/docs/src/app/fn/component-docs/generic-tag/examples/generic-tag-example/generic-tag-example.component.html

```diff
@@ -0,0 +1,35 @@
+<div fn-generic-tag>
+    <div fn-generic-tag-icon glyph="sys-enter-2"></div>
+    <div fn-generic-tag-text>Neutral</div>
+    <div fn-generic-tag-number>42K</div>
+</div>
```

Comment:
After what we discussed today I think this should become a component and the icon, text and number should be props.